addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15919,7 +15929,7 @@ msgstr ""
 #. label of a setting for the stereoscopic 3D mode of the GUI that is/should be applied
 #: system/settings/settings.xml
 msgctxt "#36500"
-msgid "Stereoscopic 3D mode (current)"
+msgid "Current stereoscopic 3D mode"
```

Comment:
I'm sorry, but this change doesn't fit the usecase of the label as it's both, a label for a setting that allows to change the stereoscopic mode as well as showing the currently applied one. The (current) part could be dropped but it would be more confusing I guess. I'd stick with what we have for now - this will most likely be reworked a bit anyways.